### ISSUE TITLE
Test and validation accuracies

### ISSUE THREAD

**Issue by marijnl** [KevinMusgrave/powerful-benchmarker#32]
Im having difficulty understanding the experiment logs of bayesian hyperparameter optimization.

I want to verify:
 * Files and dirs named `bayes_opt_*` report TEST accuracies
 * `<experiment_name><ts><trial number>` subdirs which each contain Test50_50_Partitions<num partitions>_<partition index> report VAL accuracies on each partition

**Comment by KevinMusgrave** [comment]
```bayes_opt_record_keeper_logs``` and ```bayes_opt_tensorboard_logs``` contain each the average validation accuracy of each trial. Each row will have the average validation accuracy and the associated hyperparameters for a trial. Accuracy on the test set will only be computed at the end of bayesian optimization, and it will be located in the following places:

- In the ```saved_csvs``` folder of the best performing sub experiment
- In the ```saved_csvs``` folder of any reproductions. Experiment reproductions have folder names that end with ```reproduction[0-9]```
- In the final ```accuracy_report.yaml``` located in the top-level folder, which will have average test set accuracies and confidence intervals.
- In the final ```accuracy_report_detailed.yaml``` located in the top-level folder, which will contain all test set accuracies.
- In ```bayes_opt_experiments.db```, which is located in the top level folder. I use [DB Browser](https://sqlitebrowser.org/) to easily open sqlite files and view tables.

**Comment by marijnl** [comment]
Thank you all clear. 
Maybe adding this to the `README` file would be nice. I think other users might bump into this as well

**Comment by KevinMusgrave** [comment]
Yeah once the code becomes more stable (hopefully soon) I'm planning to make a separate documentation webpage, because the README isn't easy to navigate.

**Comment by marijnl** [comment]
In found the files <best_run | reproduction>/meta_logs/saved_csvs/*TEST.csv
Great!

The only file I'm missing `accuracy_report_detailed.yaml` and `accuracy_report.yaml`

Could this be due to the warning in the logs files?
```
/anaconda/envs/py37_pytorch/lib/python3.7/site-packages/scipy/stats/_distn_infrastructure.py:1984: RuntimeWarning:

invalid value encountered in multiply

INFO:root:##### FINISHED #####
```

**Comment by KevinMusgrave** [comment]
The filename format changed since I believe v0.9.27. It's now ```report_{}.yaml``` or ```detailed_report_{}.yaml``` where ```{}``` is the description of the tester. For example: ```report_accuracies_normalized_compared_to_self_GlobalEmbeddingSpaceTester_level_0.yaml```. I made this change so that testing in different modes doesn't overwrite the same file.

**Comment by marijnl** [comment]
I see. The files `detailed_report_{}.yaml` indeed exists with v0.9.27. Thanks
